<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>511</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>PolynolialSolver V1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <widget class="QGroupBox" name="gfunc_group_box">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>371</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>G(x) data</string>
    </property>
    <widget class="QTableView" name="g_function_table_view">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>350</width>
       <height>78</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>2</width>
       <height>2</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="midLineWidth">
      <number>5</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QPushButton" name="gAddColBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>351</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rubik</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add new value</string>
     </property>
    </widget>
    <zorder>g_function_table_view</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>prompt_line_edit</zorder>
   </widget>
   <widget class="QGroupBox" name="bounds_group_box">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>260</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>2</width>
      <height>2</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>9</pointsize>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="title">
     <string>Bounds</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>351</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="from_label">
        <property name="font">
         <font>
          <family>Rubik</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>From bound   </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="from_line_edit">
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="to_label">
        <property name="font">
         <font>
          <family>Rubik</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>To bound</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="to_line_edit">
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="add_bounds_button">
        <property name="font">
         <font>
          <family>Rubik</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ffun_group_box">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>371</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>F(x) data</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QTableView" name="f_function_table_view">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>350</width>
       <height>78</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="midLineWidth">
      <number>5</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="autoScrollMargin">
      <number>0</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>2</width>
       <height>2</height>
      </size>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QPushButton" name="fAddColBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>351</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rubik</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add new value</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>340</y>
      <width>371</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Roots</string>
    </property>
    <widget class="QTextEdit" name="roots_text_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>351</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>481</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="title">
     <string>Plot Solution</string>
    </property>
    <widget class="QLineEdit" name="prompt_line_edit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>18</y>
       <width>461</width>
       <height>442</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Rubik</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::NoEcho</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>441</width>
       <height>421</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="graphical_layout"/>
    </widget>
    <zorder>gridLayoutWidget</zorder>
    <zorder>prompt_line_edit</zorder>
    <zorder>prompt_line_edit</zorder>
    <zorder>gridLayoutWidget</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>879</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rubik</family>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionGenerate_report"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionBuild"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="font">
     <font>
      <family>Rubik</family>
     </font>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGetHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="font">
    <font>
     <family>Rubik</family>
     <pointsize>9</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="font">
    <font>
     <family>Rubik</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actionGenerate_report">
   <property name="text">
    <string>Generate as HTML</string>
   </property>
   <property name="font">
    <font>
     <family>Rubik</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="font">
    <font>
     <family>Rubik</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actionBuild">
   <property name="text">
    <string>Build</string>
   </property>
   <property name="font">
    <font>
     <family>Rubik</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="font">
    <font>
     <family>Rubik</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actionGetHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
